synapse/storage/databases/main/group_server.py

```diff
@@ -766,7 +784,7 @@ def _add_room_to_summary_txn(
                     "category_id": category_id,
                     "room_id": room_id,
                 },
-                values=to_update,
+                updatevalues=to_update,
```

Comment:
This is alarming---is this a bugfix? If so, any idea why we haven't noticed sooner?

Reply:
My thought was whether it is dead code that is never called?

Reply:
I guess so. Still, good spot!